frontend/src/app/api/api/compliance.service.ts

```diff
@@ -0,0 +1,185 @@
+        if (id === null || id === undefined) {
+            throw new Error('Required parameter id was null or undefined when calling showHospitalCompliance.');
+        }
+
+        let headers = this.defaultHeaders;
+
+        // authentication (h2ms_auth) required
+        if (this.configuration.accessToken) {
+            const accessToken = typeof this.configuration.accessToken === 'function'
+                ? this.configuration.accessToken()
+                : this.configuration.accessToken;
+            headers = headers.set('Authorization', 'Bearer ' + accessToken);
+        }
+
+        // to determine the Accept header
+        const httpHeaderAccepts: string[] = [
+            'application/json'
+        ];
+        const httpHeaderAcceptSelected: string = this.configuration.selectHeaderAccept(httpHeaderAccepts);
+        if (httpHeaderAcceptSelected !== undefined) {
+            headers = headers.set('Accept', httpHeaderAcceptSelected);
+        }
+
+        // to determine the Content-Type header
+        const consumes: string[] = [];
+
+        return this.httpClient.get<any>(`${this.basePath}/hospital/${encodeURIComponent(String(id))}/compliance`,
+            {
+                headers: headers,
+                withCredentials: this.configuration.withCredentials,
+            }
+        );
+    }
+
+    /**
+     * Show a hospital ward&#39;s compliance report
+     * Returns details about a hospital ward&#39;s compliance
+     * @param hospitalId Hospital id to show
+     * @param id Ward id to show
+     */
+    public showHospitalWardCompliance(hospitalId: number, id: number): Observable<ComplianceReport> {
+        if (hospitalId === null || hospitalId === undefined) {
+            throw new Error('Required parameter hospitalId was null or undefined when calling showHospitalWardCompliance.');
+        }
+        if (id === null || id === undefined) {
+            throw new Error('Required parameter id was null or undefined when calling showHospitalWardCompliance.');
+        }
+
+        let headers = this.defaultHeaders;
+
+        // authentication (h2ms_auth) required
+        if (this.configuration.accessToken) {
+            const accessToken = typeof this.configuration.accessToken === 'function'
+                ? this.configuration.accessToken()
+                : this.configuration.accessToken;
+            headers = headers.set('Authorization', 'Bearer ' + accessToken);
+        }
+
+        // to determine the Accept header
+        const httpHeaderAccepts: string[] = [
+            'application/json'
+        ];
+        const httpHeaderAcceptSelected: string = this.configuration.selectHeaderAccept(httpHeaderAccepts);
+        if (httpHeaderAcceptSelected !== undefined) {
+            headers = headers.set('Accept', httpHeaderAcceptSelected);
+        }
+
+        // to determine the Content-Type header
+        const consumes: string[] = [];
```

Comment:
Is this being used?

Reply:
All these services are automatically generated  (see top comment) from the swagger autodocs I did in https://github.com/2018swecapstone/h2ms/pull/6, I don't think we can change it